tutorials-v5/quantum-system/JCM_Tutorial.md

```diff
@@ -0,0 +1,532 @@
+
+    # Measurement operators
+    measure_ops = [
+        jc_thermal.operators["n_c"],  # Cavity photons
+        jc_thermal.operators["sigma_plus"]
+        * jc_thermal.operators["sigma_minus"],  # Atomic excitation
+    ]
+
+    # Solve master equation
+    result = mesolve(
+        jc_thermal.hamiltonian, psi0, tlist, jc_thermal.c_ops, e_ops=measure_ops
+    )
+
+    # Plot steady-state approach
+    plt.subplot(2, 2, i + 1)
+    plt.plot(tlist, result.expect[0], "b-", label="Cavity photons", linewidth=2)
+    plt.plot(tlist, result.expect[1], "r--", label="Atomic excitation", linewidth=2)
+    plt.title(f"Thermal photons n_th = {n_th:.1f}")
+    plt.xlabel("Time")
+    plt.ylabel("Population")
+    plt.legend()
+    plt.grid(True, alpha=0.3)
+
+    # Show steady-state values
+    ss_cavity = result.expect[0][-1]
+    ss_atom = result.expect[1][-1]
+    plt.text(
+        0.8 * tlist[-1],
+        0.8,
+        f"SS cavity: {ss_cavity:.3f}\nSS atom: {ss_atom:.3f}",
+        bbox=dict(boxstyle="round,pad=0.3", facecolor="lightblue", alpha=0.7),
+    )
+
+plt.tight_layout()
+plt.show()
+```
+
+The thermal analysis demonstrates that increasing temperature leads to higher steady-state cavity occupation, with the system reaching thermal equilibrium with its environment. At high thermal photon numbers ($n_{th} = 1.0$), the steady-state cavity population approaches the thermal value, while atomic excitation remains finite due to thermal activation.
+
+
+## Vacuum Rabi Splitting
+
+One of the most important phenomena in cavity QED is vacuum Rabi splitting, the splitting of energy levels due to strong coupling between atom and cavity, even when no photons are present. To observe this phenomenon clearly, we use spectroscopic techniques to examine the power spectrum of cavity emission. We set the system parameters in appropriate units with $\omega_c = \omega_a = 2\pi$ (natural frequency units), moderate coupling $g = 0.05 \times 2\pi$, and include realistic dissipation rates to model experimental conditions. We also add thermal photons ($n_{th} = 0.25$) to simulate finite temperature effects that are always present in real experiments.
+
+In the strong coupling regime ($g > \kappa, \gamma$), the energy eigenstates of the Jaynes-Cummings model are no longer simply atomic or photonic states, but rather dressed states (polaritons) that are superpositions of both.
+
+For the resonant case, the energy levels split by $2g$ due to the atom-cavity interaction, even in vacuum. This is a purely quantum mechanical effect with no classical analog.
+
+```python
+jc = quantum_systems.jaynes_cummings(
+    omega_c=1.0 * 2 * np.pi,  # Cavity frequency
+    omega_a=1.0 * 2 * np.pi,  # Atomic frequency
+    g=0.05 * 2 * np.pi,  # Coupling strength
+    n_cavity=5,  # Small Hilbert space for clarity
+    cavity_decay=0.005,
+    atomic_decay=0.05,
+    thermal_photons=0.25,
+)
+
+# Compute Correlation Function and Spectrum
+tlist = np.linspace(0, 100, 5000)
```

Comment:
slightly increase this to 'tlist = np.linspace(0, 1000, 5000)', gives a better fit